### PR TITLE
Build/Program

### PR DESCRIPTION
**Pull request by kschzt**

### CODE REVIEW

File: browser/scripts/ui-breadcrumb.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
+/**
```
Comment (kkananen) [inline browser/scripts/ui-breadcrumb.js]: why not camelcase naming in filename?

File: browser/scripts/preset-manager.js [hunk 2]
```diff
@@ -121,6 +153,14 @@ PresetManager.prototype.openPreset = function(name) {
 	})
 }
 
+PresetManager.prototype.addObject3d = function(category, title, path) {
```
Comment (kkananen) [inline browser/scripts/preset-manager.js]: for now this is fine but in future a tag-based system might be more scalable, where object3ds (and other types) are added like other presets but with a 'object3d' tag and the list searches could be filtered to return only objects with certain tags.

**Comment by kkananen** [comment]
lgtm

in future we might want to consider a refactor where presets have arbitrary tags rather than specific object3d properties, especially if we add more categories than object3d/non-object3d.

File: browser/scripts/preset-manager.js (hunk 2 above)
Comment (kschzt) [inline browser/scripts/preset-manager.js]: agreed - let's refactor in the next iteration

**Pull request merged**